docs/INSTALL.md

```diff
@@ -30,16 +30,12 @@ Ensure that [Xcode](https://itunes.apple.com/us/app/xcode/id497799835?mt=12) is
 Then install the minimum dependencies for BLT. The preferred method is via Homebrew, though you could install these yourself without a package manager.
 
         /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-        brew install php71 git composer
+        brew install php71 git composer drush
```

Comment:
Is Drush an actual requirement, though? I thought it was only necessary if you want to run Drush commands outside the VM.